libs/withdraws/src/lib/withdraw-form.tsx

```diff
@@ -70,13 +70,14 @@ const WithdrawDelayNotification = ({
   return (
     <Notification
       intent={Intent.Warning}
+      key={symbol}
       testId={
-        threshold.isFinite()
-          ? 'amount-withdrawal-delay-notification'
-          : 'withdrawals-delay-notification'
+        threshold.isEqualTo(0)
+          ? 'withdrawals-delay-notification'
+          : 'amount-withdrawal-delay-notification'
       }
       message={[
-        !threshold.isFinite()
```

Comment:
threshold is set to Infinity if it equals 0

https://github.com/vegaprotocol/frontend-monorepo/blob/develop/libs/web3/src/lib/use-get-withdraw-threshold.tsx#L28

So in theory it should work fine